http/oidc/src/main/java/org/wildfly/security/http/oidc/OidcClientConfiguration.java

```diff
@@ -323,9 +331,13 @@ public String getTokenUrl() {
         return tokenUrl;
     }
 
-    public String getLogoutUrl() {
```

Comment:
Is this rename needed?

Reply:
The value being return is the path component of a URL not the URL itself.
For accuracy and clarity the name was changed.   In addition this variable
is directly related to the property name logout-path as defined in the 
proposal doc and the elytron-oidc-client schema.

Reply:
Sorry, I was asking about the getLogoutUrl method being renamed to getEndSessionEndpointUrl.

Reply:
I see now that getEndSessionEndpointUrl wasn't part of your commits, it was actually part of Pedro's initial changes. That's fine.